### PR TITLE
Add share component

### PR DESCRIPTION
**Pull request by jordanwade**
Implements share component to share with Facebook, Twitter, email, and sms on Event and Content items.

image
image

### CODE REVIEW

**Review by rkdavidson — DISMISSED** [review]
Tested on a couple random events and content items and only issue was for the [Tune In Online](http://local.christfellowship.church:3000/events/tune-in-online) event, the URL had a weird `#0` at the end.
Didn't look too closely, and may be harmless, but wanted to mention it.

Not sure if it's worth using some `Button` element instead of `<a>` with the useless `href` or not. Up to you!

image

**Review by drewbarontini — APPROVED** [review]
Same comment as @rkdavidson with the `#0` at the end of the URL 👍

File: components/Share/Share.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const Share = props => {
       <List py="xs" space="0">
         <Box as="li">
           <Menu.Link
-            href="#0"
+            cursor="pointer"
```
Comment (drewbarontini) [inline components/Share/Share.js:82]: We should probably put `cursor: pointer` as a CSS style in https://github.com/christfellowshipchurch/web-app-v2/blob/main/ui-kit/Menu/Menu.styles.js#L34

Comment (jordanwade) [reply components/Share/Share.js]: 👍

**Review by drewbarontini — APPROVED** [review]

**Pull request merged**